src/app.js

```diff
@@ -46,7 +46,8 @@ router.post('/courses', coursesController.addCourse);
 // router.post('/courses/{courseId}/guides/{guideId}/exercises', coursesController.getCourses);
 
 // // Exercises Service has the rest of the logic
-// router.delete('/courses/{courseId}/guides/{guideId}/exercises/{exerciseId}', coursesController.getCourses);
+// router.delete('/courses/{courseId}/guides/{guideId}/exercises/{exerciseId}',
+// coursesController.getCourses);
 
```

Comment:
remember to add the authMiddleware to check that the user is logged.
you should do something like this branch (that I am still developing) https://github.com/math-learning/exercises-service/blob/chores/zube-62-exercise-service/src/middlewares/authMiddleware.js